.CI/common.groovy

```diff
@@ -281,7 +282,7 @@ def tagName() {
 
 def makeCommand() {
   // OSX uses gmake as the GNU make program
-  return env.GMAKE ?: "make"
+  return env.GMAKE ?: "${env.MACPORTS}/bin/gmake"
```

Comment:
This needs env.MACPORTS to be available rather than GMAKE

Reply:
To be honest I'm a little lost with the Jenkins files. Could you try to fix the problem?
I will reset the latest commit.

Reply:
I've been busy grading term papers and cleaning up before vacation. I'll change GMAKE on the slaves to the macports path instead.

Reply:
Okay. Thanks a lot